tests/Aspire.Pomelo.EntityFrameworkCore.MySql.Tests/ConformanceTests.cs

```diff
@@ -13,12 +13,13 @@
 
 namespace Aspire.Pomelo.EntityFrameworkCore.MySql.Tests;
 
-public class ConformanceTests : ConformanceTests<TestDbContext, PomeloEntityFrameworkCoreMySqlSettings>
+public class ConformanceTests : ConformanceTests<TestDbContext, PomeloEntityFrameworkCoreMySqlSettings>, IClassFixture<MySqlContainerFixture>
 {
-    // in the future it can become a static property that reads the value from Env Var
-    protected const string ConnectionString = "Server=localhost;User ID=root;Password=pass;Database=test";
-
-    private static readonly Lazy<bool> s_canConnectToServer = new(GetCanConnect);
+    private readonly MySqlContainerFixture _containerFixture;
+    protected string ConnectionString => RequiresDockerTheoryAttribute.IsSupported
+                                            ? _containerFixture.GetConnectionString()
+                                            : "Server=localhost;User ID=root;Password=pass;Database=test";
+    protected readonly string ServerVersion = $"{MySqlContainerImageTags.Tag}-{MySqlContainerImageTags.Image}";
```

Comment:
```suggestion
    protected readonly string ServerVersion = $"{MySqlContainerImageTags.Tag}-mysql";
```

This isn't the Image name. Yes they happen to be the same, but it's not what this value is. There are different flavors of MySql databases (ex. MariaDB). This string tells the EF library which kind of database to prepare statements for.